app/styles/_branding.scss

```diff
@@ -0,0 +1,18 @@
+.ember-application {
+    --primary-color: $color-select;
+    --secondary-color: $color-bg-gray-blue;
+    --navbar-logo-img-url: '';
+    --hero-logo-img-url: '';
+    --hero-background-img-url: '';
+}
+
+.brand-container {
+    .branded-logo {
+        background-image: var(--navbar-logo-img-url);
+        width: 35px;
+        height: 35px;
+        margin-right: 5px;
+        background-size: 100%;
+        background-repeat: no-repeat;
+    }
+}
```

Comment:
This seems like an anti-pattern. Perhaps we can use `:global` somehow? https://github.com/css-modules/css-modules#exceptions

Reply:
let's put all the navbar branding logic/styles in the [`registries-navbar`](https://github.com/robustdj/ember-osf-web/blob/ENG-1617_update-osf-link/lib/registries/addon/components/registries-navbar/template.hbs) component, so then these styles can be a `local-class` -- `registries-navbar` can use the `brand` service and apply `{{with-branding this.brand.currentBrand}}` itself, so we don't have to change where/how the navbar component is invoked (but aren't making it harder to do so eventually).

Reply:
Does this mean that we'll be applying the `with-branding` modifier to any element that we want branded? I thought that we wanted to go with the approach of applying the modifier on some parent element and then having nested components pick up the branding?

Reply:
we should apply the modifier to some parent element instead of each branded element, yes, but `registries-navbar` is invoked in only one place -- we can either:
1. stick firm to the "modifier on some parent element" plan and move the navbar invocation down one route layer
2. leave the navbar invocation how it is, update the component to get the brand via service and set with `{{with-branding}}`

i've been assuming path 2 because it seems like changing how the navbar is invoked could cause scope creep as it affects page layouts and such, but path 1 does seem better in a lot of ways

Reply:
oh also, the navbar currently has inconsistent structure across providers (OSF brand has a dropdown with other OSF services, non-OSF brands just a logo) -- i don't think css vars will get us all the way there in a clean way (though am open to suggestions), so `registries-navbar` is going to need special cased branding logic, in any case